share/docker/run-tests.sh

```diff
@@ -15,6 +15,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+headline(){
+  echo -e "\e[1;34m#################################################################"
+  echo -e "##### $1 \e[1;37m"
+  echo -e "\e[1;34m#################################################################\e[0m"
+}
+
 set -e
 
-./build.sh test
+for lang in /avro/lang/*/
```

Comment:
The original version was broken. It effect was just doing `./build.sh` once. Here is why:
1. The current working directory is already `avro` and hence `/avro/lang/*` does not expand to anything it remains just that.
1. `cd` does not work either. Even if it worked, it would have had no effect because `cd` and the following `./build.sh` would have been executed by two different shells. So the correct way is `(cd "$lang"; ./build.sh test)`.

If you want evidence of the above, please insert `set -x` after `set -e` and see.

So you need to fix all these or just do what was actually doing in a single line.

Reply:
@thiru-apache When run via `build.sh docker-test` this script fails in its current form on master because it is running from the `/root` directory of the container where there is no `build.sh` script.

Changing to `/avro/build.sh test` would work as expected.

The reverted version in this PR also "works" since it is using absolute paths to `/avro/lang`, but it fails when it executes because it is not skipping the `c++` directory.